launchable/commands/split_subset.py

```diff
@@ -180,18 +211,44 @@ def run(self):
                         err=True)
                     return
 
-            if len(output) == 0:
-                click.echo(click.style("Error: no tests found in this subset id.", 'yellow'), err=True)
-                return
+            if is_split_by_groups:
+                group_names = []
+                for group in split_groups:
+                    group_name = group.get("groupName", "")
+                    subset = group.get("subset", [])
+                    rests = group.get("rest", [])
+
+                    if is_observation:
+                        subset, rests = subset + rests, []
+                    if is_output_exclusion_rules:
+                        subset, rests = rests, subset
+
+                    if len(subset) > 0:
+                        self.write_file("{}/subset-{}.txt".format(split_by_groups_output_dir, group_name), subset)
+                        if group_name != SPLIT_BY_GROUPS_NO_GROUP_NAME:
+                            group_names.append(group_name)
+
+                    if len(rests) > 0:
+                        self.write_file("{}/rest-{}.txt".format(split_by_groups_output_dir, group_name), rests)
```

Comment:
This is not aligned with subset https://github.com/launchableinc/cli/blob/main/launchable/commands/subset.py#L184-L192